static/js/admin/src/ResultObject.js

```diff
@@ -60,6 +66,38 @@ define('ResultObject', [ 'jquery', 'jquery-ui', 'underscore', 'ModalLoadingOverl
 		if (this.options.selected || this.selected)
 			this.select();
 	};
+
+	ResultObject.prototype._tagText = function(tag) {
+		var helpText;
+
+		switch (tag) {
+			case 'deleted':
+				helpText = 'Object has been marked for deletion'
+				break;
+			case 'described':
+				helpText = 'Object has a MODS description';
+				break;
+			case 'embargoed':
+				helpText = 'Object has an active embargo set';
+				break;
+			case 'patron-settings':
+				helpText = 'Patron access settings for this object have been added';
+				break;
+			case 'primary-object':
+				helpText = 'This file is the representative object for the work which contains it';
+				break;
+			case 'public-access':
```

Comment:
Based on the list from the ticket, I don't think `public-access` should be showing as a tag, only as a facet